### DISCUSSION TITLE
Font Embed issues after certain SDK versions

### DISCUSSION THREAD

**Discussion by koreaflash** [airsdk/Adobe-Runtime-Support#1372] [Q&A]
IDE: Flash Builder 4.7
OS: Windows 10
Build: Desktop-Window / Android

Up to Air SDK version 29.0.0.122, the embedAsCFF value is set normally and the font is applied normally.

Since the Air SDK version 31.0.0.96, the embedAsCFF value is not set properly, so a warning occurs as shown below and the font is not applied.

warning: incompatible embedded font 'Gotham' specified for spark.components::RichText (RichText262) . This component requires that the embedded font be declared with embedAsCFF=true.
warning: incompatible embedded font 'Gotham' specified for spark.components::Label (Label252) . This component requires that the embedded font be declared with embedAsCFF=true.

[Font Insertion Code - mxml]
<fx:Script>
		<![CDATA[

[Embed(source="/assets/new/fonts/GOTHAMBOLD_0.TTF", fontFamily="Gotham", fontWeight="bold", fontStyle="normal",
 embedAsCFF="true", mimeType="application/x-font-truetype")]
private var GothamB:Class;

[Embed(source="/assets/new/fonts/GOTHAMMEDIUM_2.TTF", fontFamily="Gotham", fontWeight="medium", fontStyle="normal",
 embedAsCFF="true", mimeType="application/x-font-truetype")]
private var GothamM:Class;

	]]>
</fx:Script>

Any solution?

**Comment by koreaflash** [comment]
I solved it by exporting the fonts as swc files and including them in the project.

**Comment by 2jfw** [comment]
We also had this issue and I can confirm that embedding into SWC was the **only** way to have the embedded Fonts work.
